lib/Widgets/DynamicNotebook.vala

```diff
@@ -668,6 +695,45 @@ namespace Granite.Widgets {
         // Use temporary field to avoid breaking API this can be dropped while preparing for 0.4
         string _add_button_tooltip;
 
+        public string add_button_tooltip_markup {
```

Comment:
I don't think we want to make the entire markup open-ended do we? I think we just want to set accelerators

Reply:
I've now created separated tooltip text and accels properties. Making it consistent with the current close button tooltip. Do you think that's better?